main/config/settings/.defaults.yml

```diff
@@ -173,3 +173,6 @@ createServiceCatalogPortfolio: true
 
 # Metadata provided with AWS SDK calls
 customUserAgent: 'AwsLabs/SO0144/${self:custom.settings.versionNumber}'
+
+# Available tags: https://github.com/awslabs/open-data-registry/blob/main/tags.yaml
```

Comment:
Probably worth mentioning that updating the filter doesn't result in unmounting of studies from existing workspaces

Reply:
added